testutil/testmigration/fixtures_recoverable_account.go

```diff
@@ -189,10 +200,11 @@ type ApplicationStakesConfigFn func(
 // ValidAccountsConfig defines the number of valid accounts to generate
 // for each account type in the test fixtures.
 type ValidAccountsConfig struct {
-	NumAccounts       uint64 // Number of regular externally owned accounts
-	NumApplications   uint64 // Number of application accounts
-	NumValidators     uint64 // Number of validator accounts
-	NumModuleAccounts uint64 // Number of module accounts
+	NumAccounts               uint64 // Number of regular externally owned accounts
+	NumApplications           uint64 // Number of application accounts
+	NumValidators             uint64 // Number of validator accounts
+	NumNonCustodialValidators uint64 // Number of non-custodial validator accounts
```

Comment:
_Not actionable._

---

Doesn't matter at this point but "Nodes" would've been the best term.

It encompasses both Validators and Suppliers.

I realize we're being consistent with Morse source code terminology.